terms-of-service.md

```diff
@@ -210,23 +214,23 @@ services provided by Stripe.
 ## 5. Privacy and Personal Data
 
 When delivering the Service, Whereby will collect and process
-personal data about you and your use of the Service. The Privacy
-Policy is available on [this
-page](https://whereby.com/information/tos/privacy-policy).  By using our
+personal data about you and your use of the Service. By using the
 Service, you agree that Whereby can use your personal data in
 accordance with our Privacy Policy and in accordance with the consents
-you have given us.
+you have given us. The Privacy
+Policy is available on [this
+page](https://whereby.com/information/tos/privacy-policy). 
```

Comment:
There is not a 1-to-1 mapping between this repo and our links. So we might need to introduce some extra metadata to each of these files so that'll work fine.